apps/aecore/lib/aecore/structures/data_tx.ex

```diff
@@ -44,10 +44,15 @@ defmodule Aecore.Structures.DataTx do
   use ExConstructor
 
   @spec init(tx_types(), payload(), binary(), integer(), integer()) :: DataTx.t()
-  def init(type, payload, sender, fee, nonce) do
+  def init(type, payload, sender, fee, nonce) when sender == nil or byte_size(sender) == 33 do
```

Comment:
We shouldn't hardcode pubkey size like that. The constant should be defined somewhere.

Reply:
https://en.wikipedia.org/wiki/Magic_number_(programming)